run.sh

```diff
@@ -99,7 +99,7 @@ function tear_down() {
 }
 
 function run_tests() {
-   eval "./mvnw -Dtest=$TESTS  test"
+   eval "./mvnw -Dtest=$TESTS -DPLATFORM_TYPE=$PLATFORM test"
```

Comment:
that's correct, have to set that manually if I'm skipping the setup part when using CF, local works fine with the default